app/workers/sha_poller_worker.rb

```diff
@@ -0,0 +1,17 @@
+# Polls weather for subscribed locations
+class ShaPollerWorker
+  include Sidekiq::Worker
+  include Sidekiq::Repeat::Repeatable
+
+  repeat { hourly.minute_of_hour(0, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55) }
+
+  def perform
+    logger.info "Running sha poll"
+
+    Helios2Schema.subscriptions.trigger(
+      "schemaShaPublished",
```

Comment:
Oh I see, the trigger function must do a lookup into the schema subscriptions. Why do we add `Published` onto the end here? I would expect it to map more directly to the schema field name `schemaSha`.

Reply:
I initially wrote `published` at the end to keep consistency with the other subscription types and thought it was a proper formatting when working off `weather_poller_worker.rb`. I changed the field name from `schemaSha` to `deployment_sha` since it is the "deployment" sha, per Craig's note above.

Reply:
I found the documentation for this here: https://graphql-ruby.org/subscriptions/triggers.html

> Events are triggered by name, and the name must match fields on your Subscription Type